inst/templates/mapping_template_NAVIGATE.csv

```diff
@@ -161,25 +161,25 @@ idx;Tier;Category;Variable;Unit;piam_variable;piam_unit;piam_factor;internal_com
 132;2;demography;Population|Rural;million;;;;;;Total population living in rural areas;
 133;2;demography;Population|Urban;million;Population|Urban;million;;;;Total population living in urban areas;x
 134;2;demography;Population|Working Age;million;;;;;;Total working age population (age 15-65 years);
-135;1;economy;Consumption;billion US$2010/yr;Consumption;billion US$2005/yr;1.10774;;;total consumption of all goods, by all consumers in a region;
```

Comment:
Honestly, I think this is wrong. Probably, you opened the file in excel or so with wrong settings, and it somehow transformed the values incorrectly. Please revert the changes from 1.10774 to 110.774 back to the old value…